src/bin/wasmer.rs

```diff
@@ -683,8 +687,12 @@ fn execute_wasm(options: &Run) -> Result<(), String> {
                 args.push(Value::I32(x));
             }
 
+            let invoke_fn = match options.invoke.as_ref() {
+                Some(fun) => fun,
+                _ => "main",
+            };
```

Comment:
If you don't take the above advice:

⛳️ 
```Rust
let invoke_fn = options.invoke.as_ref().unwrap_or("main");
```

Reply:
```suggestion
            let invoke_fn = options.invoke.as_ref().unwrap_or("main");
```

Reply:
The problem with suggestions is that they only apply to 1 line and standard Rust fmt encourages splitting things across many lines.  It'd be extremely useful if they could apply to multiple lines and be batch applied; like a merge request of a PR but done inline